easybuild/easyblocks/c/clang.py

```diff
@@ -197,35 +198,47 @@ def configure_step(self):
 
     def disable_sanitizer_tests(self):
         """Disable the tests of all the sanitizers"""
-        patchfiles = [
-            "projects/compiler-rt/lib/asan/CMakeLists.txt",
-            "projects/compiler-rt/lib/dfsan/CMakeLists.txt",
-            "projects/compiler-rt/lib/lsan/CMakeLists.txt",
-            "projects/compiler-rt/lib/msan/CMakeLists.txt",
-            "projects/compiler-rt/lib/tsan/CMakeLists.txt",
-            "projects/compiler-rt/lib/ubsan/CMakeLists.txt",
-        ]
-
-        for patchfile in patchfiles:
+        if LooseVersion(self.version) < LooseVersion('3.6'):
+            patchfiles = [
+                "projects/compiler-rt/lib/asan/CMakeLists.txt",
+                "projects/compiler-rt/lib/dfsan/CMakeLists.txt",
+                "projects/compiler-rt/lib/lsan/CMakeLists.txt",
+                "projects/compiler-rt/lib/msan/CMakeLists.txt",
+                "projects/compiler-rt/lib/tsan/CMakeLists.txt",
+                "projects/compiler-rt/lib/ubsan/CMakeLists.txt",
```

Comment:
flesh out the common part `projects/compiler-rt`, throw that in the `os.path.join` below`